pyenvbuilder/commands.py

```diff
@@ -0,0 +1,23 @@
+'''
+pyenvbuilder's commands
+'''
+
+import argparse
+import sys
+import pyenvbuilder
+
+def add_commands():
+    '''
+    Defines shell commands
+    '''
+
+    parser = argparse.ArgumentParser(description='Python Environment Builder')
+
+    parser.add_argument('--V', '--version', action='version',
```

Comment:
I just realized this, but I think i should change the option `--V` to just `-V` - single dash

Reply:
Why not just --version?

Reply:
Yeah that sounds good :)